query-languages/m/type-replacetablekeys.md

```diff
@@ -13,4 +13,62 @@ Type.ReplaceTableKeys(<b>tableType</b> as type, <b>keys</b> as list) as type
 
 ## About
 
-Returns a new table type with all keys replaced by the specified list of keys.
+Returns a new table type with all keys replaced by the specified list of keys. 
+
+Each key is defined using a record in the following form: 
+* `Columns`—a list of the column names that define the key
```

Comment:
For lists like this, we usually use colon followed by a space.

```suggestion
* `Columns`: a list of the column names that define the key
```

The same would apply to the other bullets as well.